packages/nx-container/src/executors/build/context.ts

```diff
@@ -72,8 +72,8 @@ export async function getInputs(
     buildArgs: await getInputList('build-args', prefix, options['build-args'], true),
     buildContexts: await getInputList('build-contexts', prefix, options['build-contexts'], true),
     builder: core.getInput('builder', { prefix, fallback: options.builder }),
-    cacheFrom: await getInputList('cache-from', prefix, options['cache-from'], true),
-    cacheTo: await getInputList('cache-to', prefix, options['cache-to'], true),
+    cacheFrom: await getInputList('cache-from', prefix, [options['cache-from']].flat(), true),
+    cacheTo: await getInputList('cache-to', prefix, [options['cache-to']].flat(), true),
```

Comment:
Can you validate the existence of options['cache-from'] and options['cache-to'], otherwise you can end with and `[undefined]` array

Reply:
Should be fixed. I have added fallback empty array.